### PR TITLE
Feature/aboutus

### PR DESCRIPTION
**Pull request by Margaret-Nyambura**
# Engineers Checklist

## What this PR does?
- List of updates for features, enhancements, or bugfixes

## How can it be tested?
- Please provide step-by-step guide on how to test
- If web or mobile, screenshots will be very helpful in addition

# Where is it documented?
- List JIRA tickets

## Backend Updates
- [ ] Updated Postman Documentation
	- [ ] Requests
	- [ ] Examples
	- [ ] Tests
- [ ] Updated Tasks on JIRA
- [ ] Tested API updates in `dev` environment
- [ ] CI Checks Passed

### CODE REVIEW

**Review by Susan-Favour — APPROVED** [review]

**Review by Maureen-Wangari — DISMISSED** [review]

**Review by Wato-Adano — DISMISSED** [review]

**Review by priscilla-mikisa — APPROVED** [review]

**Review by Maureen-Wangari — DISMISSED** [review]

**Review by Susan-Favour — APPROVED** [review]

File: maka-agile-app/src/About copy/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+const AboutCopy = ()=>{
+    return(
+        <div className="container">
+            <div className="about-text">
+
+            <p>We are an online learning platform aimed to bring together parents, teachers, and students, 
+    specifically aimed at well-resourced primary schools in Kenya. 
+    This platform will offer live classes attended by all students, led by their teachers.</p>
+    
+            <h3 className="head"><u>Mission</u></h3>
```
Comment (hunterachieng) [inline maka-agile-app/src/About copy/index.js:11]: Underline can be achieved in css
Change the classname

File: maka-agile-app/src/App.js [hunk 2]
```diff
@@ -1,15 +1,18 @@
 import Team from "./Team";
 import "./index.css"
 import Services from "./Services";
+import AboutCopy from "./About copy";
```
Comment (hunterachieng) [inline maka-agile-app/src/App.js:4]: Why not rename your component? What is about copy?

File: maka-agile-app/src/Team/index.css [hunk 3]
```diff
@@ -1,30 +1,31 @@
...
+    /* display: flex; */
+    /* gap: 18%; */
+    /* padding-left: 260px; */
 }
 
 #twoImagesColumn{
   display: flex;
-  gap: 18%; 
-  padding-left: 550px;
+  /* gap: 18%;  */
```
Comment (hunterachieng) [inline maka-agile-app/src/Team/index.css:11]: Remove the commented code

File: maka-agile-app/src/AboutUs/index.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import './index.css'
+const AboutCopy = ()=>{
```
Comment (hunterachieng) [inline maka-agile-app/src/AboutUs/index.js:2]: You are still calling it AboutCopy

**Review by hunterachieng — APPROVED** [review]

**Review by priscilla-mikisa — APPROVED** [review]

**Review by Wato-Adano — APPROVED** [review]

**Review by Susan-Favour — APPROVED** [review]

**Review by Maureen-Wangari — APPROVED** [review]

**Pull request merged**